doc/classes/ProjectSettings.xml

```diff
@@ -2968,23 +2972,28 @@
 			The number of entries in the sampler descriptors heap the Direct3D 12 rendering driver uses each frame, used for most rendering operations.
 			Depending on the complexity of scenes, this value may be lowered or may need to be raised.
 		</member>
-		<member name="rendering/rendering_device/driver" type="String" setter="" getter="">
+		<member name="rendering/rendering_device/driver" type="String" setter="" getter="" default="&quot;auto&quot;">
 			Sets the driver to be used by the renderer when using a RenderingDevice-based renderer like the Forward+ or Mobile renderers. This property can't be edited directly. Instead, set the driver using the platform-specific overrides. This can be overridden using the [code]--rendering-driver &lt;driver&gt;[/code] command line argument.
+			Supported values are:
+			- [code]auto[/code], Metal on Apple Silicon Macs and iOS, D3D12 on Windows if engine was compiled w/o Vulkan support, Vulkan in all other cases.
```

Comment:
Consoles have their own defaults so we should likely make that a bit more explicit.
```suggestion
			- [code]auto[/code], Metal on Apple Silicon Macs and iOS, Vulkan on other built-in platforms. On Windows, Direct3D 12 is the default if the engine was compiled without Vulkan support.
```